File: URI-py/1165.py
# -*- coding: utf-8 -*-
n = int(input())
for i in range(n):
    primo = True
    v = int(input())
    for l in range(2,v):
        if (v%l)==0:
            primo = False
            break
    if (primo) == True:
        print(v,"eh primo")
    else:
        print(v,"nao eh primo")

File: URI-cpp/1165.cpp
#include <iostream>

using namespace std;

int main()
{
    int n,m;
    cin >> n;
    for (int i = 0; i < n ; ++i) {
        bool validade = true;
        cin >> m;
        for (int j = 2; j < m; ++j) {
            if(m%j==0){
                validade = false;
                break;
            }
        }
        if (!validade) {
            printf("%d nao eh primo\n",m);
        } else {
            printf("%d eh primo\n",m);
        }
    }
    return 0;
}
